src/api.js

```diff
@@ -28,4 +28,18 @@ export const getArticleCommentsById = (article_id) => {
     // console.log(data.comments)
      return data.comments
   });
-};
\ No newline at end of file
+};
+
+export const postCommentByArticleId = (article_id, commentBody) => {
+  return axios.post(`${baseURL}/articles/${article_id}/comments`, commentBody).then(({data}) => {
+    // console.log(data.comment)
+    return data.comment;
+  })
+}
+
+export const deleteCommentById = (comment_id) => {
+  return axios.delete(`${baseURL}/comments/${comment_id}`).then(({data}) => {
+    // console.log(data.comment)
```

Comment:
Remember to clean up your unused logs :)

Reply:
Of course... I will do it!